js/data.js

```diff
@@ -1 +1,14 @@
 /* exported data */
+var data = {
+  coins: []
+};
+
+window.addEventListener('beforeunload', event => {
+  var steralData = JSON.stringify(data);
+  localStorage.setItem('cryptocurrency', steralData);
+});
+
+if (localStorage.getItem('code-journal')) {
```

Comment:
You should be grabbing "cryptocurrency" here, you never set "code-journal" as the key pair. Also, you shouldn't be naming this code-journal anyway